test/functional-portable/corerp/cloud/resources/testdata/recipes/test-terraform-recipes/azure-storage/main.tf

```diff
@@ -0,0 +1,32 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = "~> 3.0.0"
+    }
+  }
+}
+
+resource "random_id" "unique_name" {
+  byte_length = 8
+}
+
+resource "azurerm_storage_account" "test_storage_account" {
+  name = "acct${random_id.unique_name.hex}"
+  resource_group_name = var.resource_group_name
+  location = var.location
+  account_tier = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_storage_container" "test_container" {
+  name = "ctr${random_id.unique_name.hex}"
+  storage_account_name = azurerm_storage_account.test_storage_account.name
+}
+
+resource "azurerm_storage_blob" "test_blob" {
+  name = "blob${random_id.unique_name.hex}"
+  storage_account_name = azurerm_storage_account.test_storage_account.name
+  storage_container_name = azurerm_storage_container.test_container.name
+  type = "Block"
+}
```

Comment:
Is this actually new? or just a rename that github couldn't figure out?

Reply:
Nothing new here